### ISSUE TITLE
WCAG: Tab order fix when creating a new group

### ISSUE THREAD

**Issue by Febakke** [Altinn/app-frontend-react#908]
### Description of the bug

When the user click the "legg til ny" button the next element in the tab order after pressing enter is the "rediger" button on the next repeating group component. It jumps over the entire group that you just said that you wanted to add. 

Picture One is before you click "legg til ny"
![Skjermbilde 2023-02-08 kl  15 35 53](https://user-images.githubusercontent.com/33222679/217560553-1f6799bf-5bc7-418f-ac35-9a98cafad46b.png)

Picture 2 is after you pressed "Legg til ny" and tabbed once. The focus is on "rediger" button on the next component. It should be on the first input element of your newly made group. In this case: "1. Endre fra"
![Skjermbilde 2023-02-08 kl  15 36 00](https://user-images.githubusercontent.com/33222679/217560732-49dfc3c3-1160-4bbe-be6a-0afa63aef522.png)

### Steps To Reproduce

1. Go to an app with repeating groups.
2. Try to create a new one using only your keyboard.
3. You should jump over the group you just created
